--- /home/users/hunter/backup/Makefile~	Sun Dec 29 18:49:38 2002
+++ Makefile	Sun Dec 29 18:49:38 2002
@@ -12,8 +12,8 @@
 # BASE_DIR   = /usr/local
 # BASE_DIR   = /opt
 BIN_DIR   = $(BASE_DIR)/bin
-LIB_DIR   = $(BASE_DIR)/lib/cdlabelgen
-MAN_DIR   = $(BASE_DIR)/man
+LIB_DIR   = $(BASE_DIR)/share/cdlabelgen
+MAN_DIR   = $(BASE_DIR)/share/man
 SOURCES    = cdlabelgen ChangeLog INSTALL README Makefile cdinsert.pl INSTALL.WEB cdlabelgen.pod cdlabelgen.1 cdlabelgen.html cdlabelgen$(VERSION).spec
 POSTSCRIPT = template.ps *.eps
 DISTFILES = $(SOURCES) $(POSTSCRIPT)
